docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,128 @@
+---
+product: base
+title: React Modal component
+components: ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+packageName: '@mui/base'
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other elements that force the user to take action before continuing.</p>
```

Comment:
<img width="500" alt="Screen Shot 2022-03-18 at 11 38 45" src="https://user-images.githubusercontent.com/67129314/159023489-90c7e3aa-e13b-473d-b44e-8a8972ee879e.png">

Not sure why but the backticks in the description text seems to not be working 😕

Reply:
Is it because they're inside of a `<p>` tag? 

Speaking of which: why do these docs use `<p>` at the top here and then Markdown everywhere else? 🤔

Reply:
Let's not use backticks, unless specifically referring to code or props.

> why do these docs use <p> at the top here and then Markdown everywhere else

Deep in the infrastructure sources, there is code that relies on the document having a summary. The `<p class="description">` tag is a way to mark it as one.

Reply:
Thanks for that clarification @michaldudak!